src/Atmos/Model/tendencies_momentum.jl

```diff
@@ -16,6 +16,26 @@ function flux(::PressureGradient{Momentum}, m, state, aux, t, ts, direction)
     end
 end
 
+#####
+##### Second order fluxes
+#####
+
+struct ViscousStress{PV <: Momentum} <: TendencyDef{Flux{SecondOrder}, PV} end
+function flux(
+    ::ViscousStress{Momentum},
+    m,
+    state,
+    aux,
+    t,
+    ts,
+    diffusive,
+    hyperdiff,
+)
+    ν, D_t, τ = turbulence_tensors(m, state, diffusive, aux, t)
+    ν, D_t, τ = sponge_viscosity_modifier(m, m.viscoussponge, ν, D_t, τ, aux)
```

Comment:
Does this term mutate `τ`? Could we write `modify_turbulence_tensors!(τ, args...)` if so? It doesn't look like `ν, D_t` are used after this, why does `sponge_viscosity_modifier` return them?

Reply:
It does, and we used to. But they're scalars, and Julia [does not pass scalars by reference](https://github.com/CliMA/ClimateMachine.jl/issues/1638).